test/tests/fuel-cycle/fuel_cycle.i

```diff
@@ -0,0 +1,450 @@
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 0
+  []
+  [HX_CPS_leak_fraction] #f_{5-6}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1e-2
+  []
+  [HX_EXO_leak_fraction] #f_{5-10}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1e-4
+  []
+  [HX_FW_leak_fraction] #f_{5-3}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 0.6
+  []
+  [ISS_EXO_leak_fraction] #f_{9-10}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1e-1
+  []
+  [P_DIV_leak_fraction] #f_{P-4}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1e-4
+  []
+  [P_FW_leak_fraction] #f_{P_3}
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1e-4
+  []
+  [TES_efficiency] #eta_2
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 0.95
+  []
+  [TBR] #According to the PhD Thesis referenced in the paper,
+        # this is the required Tritium Breeding Ratio (TBR)
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1.511145
+  []
+  [residence1] #BZ
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 86400 #1 day, Abdou
+    #value = 8640-86400 EXOTIC-6-7-8
+  []
+  [residence10] #EXO
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 3600 # Day et al
+    #value = 72000 #Coleman et al
+  []
+  [residence2] #TES
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 86400 # Abdou
+    #value = 86400-432000 Riva
+  []
+  [residence3] #FW
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1000 #Riva
+  []
+  [residence4] #DIV
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1000 #Riva
+  []
+  [residence5] #HX
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1000 # Abdou 2021 paper chosen for analysis
+  []
+  [residence6] #CPS
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 864000 #Abdou 1,4
+    #value 864000 (analysis case)
+  []
+  [residence7] #Vac
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value = 1800 # 30 minutes (20-30 minutes)
+  []
+  [residence8] #FCU
+    type = ConstantPostprocessor
+    execute_on = 'TIMESTEP_BEGIN INITIAL LINEAR NONLINEAR'
+    value =  4680 # 4 hours as per "reasonable approx" Abdou
```

Comment:
```suggestion
    value =  14400 # 4 hours as per "reasonable approx" Abdou
```
I think you had the number from Day (1.3 hours).

Reply:
This is where I had to tweak things when making the Figure 3 comparison. As presented, Figure 3 represents that the sum of residence 7, 8 and 9 are together 4 hours, so I took the number from Day, but forgot to update the notes in the input file.

Reply:
I see that you just changed it to `1`. Why is that?

Reply:
Never mind, I see that you have changed these assumptions. All good and well documented.